src/components/table/partial-styles/tabulator-custom-styles.scss

```diff
@@ -87,6 +87,22 @@
                     }
                 }
             }
+            .recent-active {
+                &:before {
+                    $width-of-selected-row-indicator: 0.2rem;
+                    content: '';
+                    display: inline-block;
+                    box-sizing: border-box;
+                    position: sticky;
+                    z-index: $table--has-interactive-rows--selectable-row--hover +
+                        1;
+                    inset: 0 0 auto 0;
+                    border: $width-of-selected-row-indicator solid
+                        rgb(211, 211, 211);
+                    border-radius: 1rem;
+                    margin-right: -$width-of-selected-row-indicator * 2;
+                }
+            }
```

Comment:
this is mostly repetition of lines 48 to 60.
it's better to either make a `mixin` for these lines; or have a common CSS rule for both. The only difference is the colors of the dots, which can the be written in a separate CSS rule for each